src/Validators/EmailValidator.php

```diff
@@ -10,6 +10,10 @@
 use WMDE\FunValidators\DomainNameValidator;
 use WMDE\FunValidators\ValidationResult;
 
+const BANNED_DOMAINS = [
+	'example.com'
+];
```

Comment:
This violates PSR-2 if I'm not mistaken.

Why not put the const in the class?

Reply:
Yeah, that was a last minute oversight. Either way, I am closing this PR because I don't like this approach anyway, I will go another route in FundraisingFrontend directly.